hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/DeleteBlocksCommandHandler.java

```diff
@@ -482,23 +478,19 @@ private void updateMetaData(KeyValueContainerData containerData,
     }
   }
 
-  private boolean isTxnIdValid(long containerId,
+  private void logDeleteTransaction(long containerId,
       KeyValueContainerData containerData, DeletedBlocksTransaction delTX) {
-    boolean b = true;
     if (LOG.isDebugEnabled()) {
-      LOG.debug("Processing Container : {}, DB path : {}", containerId,
-          containerData.getMetadataPath());
+      LOG.debug("Processing Container : {}, DB path : {}, transaction {}",
+          containerId, containerData.getMetadataPath(), delTX.getTxID());
     }
 
     if (delTX.getTxID() <= containerData.getDeleteTransactionId()) {
-      if (LOG.isDebugEnabled()) {
-        LOG.debug(String.format("Ignoring delete blocks for containerId: %d."
-                + " Outdated delete transactionId %d < %d", containerId,
-            delTX.getTxID(), containerData.getDeleteTransactionId()));
-      }
-      b = false;
+      LOG.info(String.format("Delete blocks for containerId: %d"
+              + " is outdated, either received out of order or retried,"
+              + " %d < %d", containerId, delTX.getTxID(),
+          containerData.getDeleteTransactionId()));
```

Comment:
Suggest changing log message to have `<=` to match the condition being checked, and removing "outdated" since it may not be outdated if the deletes were received out of order.
```suggestion
      LOG.info(String.format("Delete blocks for containerId: %d"
              + " is either received out of order or retried,"
              + " %d <= %d", containerId, delTX.getTxID(),
          containerData.getDeleteTransactionId()));
```

Reply:
@errose28 updated